rust/bridge/shared/src/ffi/storage.rs

```diff
@@ -77,7 +77,10 @@ impl IdentityKeyStore for &FfiIdentityKeyStoreStruct {
         Ok(IdentityKeyPair::new(IdentityKey::new(pub_key), *priv_key))
     }
 
-    async fn get_local_registration_id(&self, ctx: Context) -> Result<u32, SignalProtocolError> {
+    async fn get_local_registration_id(
+        &self,
+        ctx: Context,
+    ) -> Result<DeviceId, SignalProtocolError> {
```

Comment:
This is *not* a device ID; it's a locally-generated random number used for pre-keys / constructing the initial session.

Reply:
Thanks! I have used that phrasing in the documentation for the new `SessionSeed` struct.